bigframes/core/compile/googlesql/query.py

```diff
@@ -51,10 +55,14 @@ class Select(abc.SQLSyntax):
 
     select_list: typing.Sequence[typing.Union[SelectExpression, SelectAll]]
     from_clause_list: typing.Sequence[FromClause] = ()
+    distinct: bool = False
 
     def sql(self) -> str:
         text = ["SELECT"]
 
+        if self.distinct:
```

Comment:
Could you please write unit tests for the new functions in this file? I'm aiming for 100% unit-test coverage for AST compilation.

Reply:
Updated.